Game/Config/DefaultSpatialGDKSettings.ini

```diff
@@ -49,8 +49,8 @@ RPCQueueWarningDefaultTimeout=2.000000
 bEnableNetCullDistanceInterest=True
 bEnableNetCullDistanceFrequency=False
 FullFrequencyNetCullDistanceRatio=0.330000
-InterestRangeFrequencyPairs=(DistanceRatio=1.000000,Frequency=0.500000)
-+InterestRangeFrequencyPairs=(DistanceRatio=0.660000,Frequency=2.000000)
+InterestRangeFrequencyPairs=(DistanceRatio=1.000000,Frequency=7.500000)
++InterestRangeFrequencyPairs=(DistanceRatio=0.660000,Frequency=15.000000)
```

Comment:
What do these numbers mean? 

< 0.66 is 15 hz and the rest is 7.5hz?

Reply:
This means all actors within 1/3 of their NCD are updated at full frequency
All actors up to 2/3 of their NCD are updated at 15hz
The remaining actors are updated at 7.5hz

Reply:
oh yeah - what you said